### ISSUE TITLE
Images in ColoredProductExtension.php   

### ISSUE THREAD

**Issue by sparkcom** [silvershop/silvershop-coloredvariations#13]
Hello, 

I propose to change 'Images' to 'ColorImages' to make it more specific. 

That's because when people extend Product.php to have more product images, more often, people may just use 'Images' for Product Images. And if that happens, there will be a conflict between this extension and an extension to have more product images.

**Comment by wilr** [comment]
@sparkcom Makes sense, would also be an issue if you added `Images` to the base Page type which could be common. If you submit a PR I'll tag a new major release and document the change. I think we need to include a migration script for people so that it automatically comes across.
